strongbox.desktop

```diff
@@ -0,0 +1,6 @@
+[Desktop Entry]
```

Comment:
what other fields may be useful?

Reply:
https://specifications.freedesktop.org/desktop-entry-spec/latest/apa.html

`Version` may be nice in future